RecoMuon/MuonIdentification/plugins/MuonIdProducer.cc

```diff
@@ -750,6 +779,16 @@ bool MuonIdProducer::isGoodRPCMuon( const reco::Muon& muon )
   return ( muon.numberOfMatchedRPCLayers( reco::Muon::RPCHitAndTrackArbitration ) > minNumberOfMatches_ );
 }
 
+bool MuonIdProducer::isGoodME0Muon( const reco::Muon& muon )
+{
+  if(muon.track()->pt() < minPt_ || muon.track()->p() < minP_) return false;
```

Comment:
minP_ and minPt_ were derived to match the lowest penetrating momentum in eta up to 2.4. 
Please check that it still applies for ME0.
The version going to eta ~4 (if it ever becomes real) will end up being cut too much.
A different set of cuts may be needed.